examples/create-machine.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const Machine = require('..');
+
+var options = {
+  "driver": "virtualbox"
+};
+
+// List all machines with additional metadata
```

Comment:
Please update this comment and the arguments of the callback, or remove the example entirely, it's not really necessary. Up to you :)